specification/electronic-prescription-service-api.yaml

```diff
@@ -133,8 +133,48 @@ info:
     We are hoping to make the onboarding process as lightweight and as self-service as possible.
 
     More details to follow.
-
-      
+    
+    ## Responses
+    ### 2xx responses
+    **Note: This section is under development and subject to change. All Spine responses are returned with 200 HTTP codes until a Spine translation service is built. This does not affect the sandbox environment.**
+    | Endpoint              | HTTP code | Response format                                                                                                                                             |
+    | --------------------- | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+    | POST /$prepare        | 200       | [See /$prepare response body](https://digital.nhs.uk/developer/api-catalogue/electronic-prescription-service-fhir#api-Prescribing-preparePrescription)      |
+    | POST /$convert        | 200       | [See /$convert response body](https://digital.nhs.uk/developer/api-catalogue/electronic-prescription-service-fhir#api-Prescribing-convertPrescription)      |
```

Comment:
not sure we wanna reference `$convert` in the spec?

Reply:
really? i believe you but how come

Reply:
convert is an endpoint for dev use / testing / 'internal' purposes and spec is public facing

Reply:
resolving as i've removed